scripts/set-git-rev/default.nix

```diff
@@ -0,0 +1,42 @@
+# This provides a function like pkgs.haskell.lib.justStaticExecutables,
```

Comment:
I'm a bit confused - which bit makes it static?

Reply:
comment coming from a copy and paste, will remove. This was originally taken from cardano stuff that combines the static things with the git stuff however we don't need the static stuff